images/openstack/Dockerfile.ci

```diff
@@ -5,11 +5,13 @@ WORKDIR /go/src/github.com/openshift/installer
 COPY . .
 RUN hack/build.sh
 
-FROM registry.svc.ci.openshift.org/origin/4.2:base
+FROM registry.svc.ci.openshift.org/origin/4.4:cli AS cli
+
+FROM registry.svc.ci.openshift.org/origin/4.4:base
+COPY --from=cli /usr/bin/oc /bin/oc
 COPY --from=builder /go/src/github.com/openshift/installer/bin/openshift-install /bin/openshift-install
 COPY --from=builder /go/src/github.com/openshift/installer/images/openstack/rdo-stein.repo /etc/yum.repos.d/rdo-stein.repo
 COPY --from=builder /go/src/github.com/openshift/installer/images/openstack/rdo-stein.gpg /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-Cloud
-COPY --from=registry.svc.ci.openshift.org/origin/4.2:cli /usr/bin/oc /bin/oc
```

Comment:
* `base` tag was `4.2`, it becomes `4.4`
* `cli` tag was `4.4`, it becomes `4.2`

It that intentional?

Reply:
This isn't the goal, I just did this to keep things up to date (other images still reference 4.2 tho).
What is preferred here, leaving it at 4.2 or adding a comment in commit msg?

Reply:
The best would be to separate the changes, so that they can be approved, tested and possibly reverted independently. Thanks!

Reply:
Updated. I'll change the version tags for all images in a follow up